packages/server/src/api/controllers/rowAction/crud.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  CreateRowActionRequest,
+  Ctx,
+  RowActionResponse,
+  RowActionsResponse,
+  UpdateRowActionRequest,
+} from "@budibase/types"
+import sdk from "../../../sdk"
+
+async function getTable(ctx: Ctx) {
+  const { tableId } = ctx.params
+  const table = await sdk.tables.getTable(tableId)
+  if (!table) {
+    ctx.throw(404)
+  }
+  return table
+}
+
+export async function find(ctx: Ctx<void, RowActionsResponse>) {
```

Comment:
Should these not use `UserCtx` instead of `Ctx`?

Reply:
I'm not sure if it makes any difference in this case. `UserCtx` will have the user typed, but nothing else, and we don't use it here at all